CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ Please add to the relevant subsections under Unreleased below on every PR where
 - Update the signature and docstring of `wandb.api.public.runs.Run.log_artifact()` to support artifact tags like `Run` instances returned by `wandb.init()`. (@tonyyli-wandb in https://github.com/wandb/wandb/pull/8414)
 - Add docstring for `wandb.watch` to support auto-complete (@kptkin in https://github.com/wandb/wandb/pull/8425)
 - Fix glob matching in define metric to work with logged keys containing `/` (@KyleGoyette in https://github.com/wandb/wandb/pull/8434)
+- Fix metric names not being able to container a dot (`.`) (@jacobromero in https://github.com/wandb/wandb/pull/8445)
```

Comment:
```suggestion
- Fix metric names not being able to contain a dot (`.`) (@jacobromero in https://github.com/wandb/wandb/pull/8445)
```

Reply:
actually, on second thought, I think this should mention `define_metric` explicitly

> Allow "a\\.b" syntax in `run.define_metric` to refer to a dotted metric name